PVM_PVP/Any/ModernSlayer.py

```
@@ -0,0 +1,493 @@
+                    if any(prop in props_lower for prop in ["slayer", "damage", "durability", "weapon", "swing", "thrust", "shoot"]):
+                        return weapon
+            except:
+                pass
+    
+    return None
+
+def is_container(item):
+    """Check if an item is a container."""
+    try:
+        API.ItemsInContainer(item.Serial)
+        return True
```

Ruff (0.13.1)

Consider moving this statement to an `else` block

(TRY300)